File: Soal-4/PRAK104-2410817310014-INDRA_SURYADILAGA.c
#include <stdio.h>

int main() {
    int hargaA = 100000, hargaB = 200000;
    float diskonA = 15, diskonB = 70;
    
    printf("Harga sepatu A adalah %d\n",hargaA);
    printf("Harga sepatu B adalah %d\n",hargaB);
    printf("Harga sepatu A setelah diskon %.0f%% adalah %.0f \n", diskonA, hargaA-hargaA*diskonA/100);
    printf("Harga sepatu B setelah diskon %.0f%% adalah %.0f \n", diskonB, hargaB-hargaB*diskonB/100);
    return 0;
}

File: Soal-4/PRAK104-2410817310014-INDRA_SURYADILAGA.py
sepA=100000
sepB=200000
dis1=15
dis2=70

charsep = ['A', 'B']    
datasep = [sepA, sepB]
datadis = [dis1, dis2]

for char, data in zip(charsep, datasep):
    print(f'Harga sepatu {char} adalah {data}')

for char, data, dis in zip(charsep, datasep, datadis):
    harga_diskon = int(data - (data * dis/100))
    print(f'Harga sepatu {char} setelah diskon {dis}% adalah {harga_diskon}')
